releasenotes/notes/deprecate-convert-ingress.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: release-notes/v2
+kind: deprecate-istioctl-cmd
+area: istioctl
+issue:
+  - 29153
+
+releaseNotes:
+- |
+  **Deprecated** istioctl convert-ingress command
```

Comment:
Generally "deprecated" means we are telling people not to use something. "Removed" means it actually removed like you did here

Reply:
ok